content/spin/http-trigger.md

```diff
@@ -35,54 +44,72 @@ by defining the top-level application trigger:
 trigger = { type = "http", base = "/" }
 ```
 
-Then, when defining the component (in `spin.toml`), there are two pieces of
-configuration that can be set for the component trigger: the route,
-and the _HTTP executor_ (see details below about executors). For example:
+The HTTP trigger also requires a `base` field.  Spin interprets each component route as relative to this route.  In most cases, you can set this to `"/"`, the base of the Web server, meaning Spin applies no prefix to component routes.
+
+> If you create an application from a HTTP template, the trigger will be already set up for you.
+
+In addition, each component must have HTTP-specific configuration in its `[component.trigger]` table.
+
+## Mapping a Route to a Component
 
-- an HTTP component configured on the `/hello` route that uses the Spin executor:
+Each component handles one route, specified in the `route` field of the component manifest.
+
+The route may be _exact_ or _wildcard_.
+
+An _exact_ route matches only the given route.  This is the default behavior.  For example, `/cart` matches only `/cart`, and not `/cart/checkout`.
 
 <!-- @nocpy -->
 
 ```toml
+# Run the `cart.wasm` module when the application receives a request to `/cart`...
+[[component]]
+id = "cart"
+source = "cart.wasm"
 [component.trigger]
-route = "/hello"
-executor = { type = "spin" }
+route = "/cart"
+
+# ...and the `checkout.wasm` module for `/cart/checkout`
+[[component]]
+id = "checkout"
+source = "checkout.wasm"
+[component.trigger]
+route = "/cart/checkout"
 ```
 
-- an HTTP component configured on the `/goodbye` route that uses the Wagi executor:
+A _wildcard_ route matches the given route and any route under it.  A route is a wildcard if it ends in `/...`.  For example, `/users/...` matches `/users`, `/users/1`, `/users/1/edit`, and so on.  Any of these routes will run the mapped component.
+
+> In particular, the route `/...` matches all routes.
```

Comment:
```suggestion
> In particular, the route `/...` matches all routes:
```

Reply:
Not here, this is an aside and the code block isn't.

Reply:
Good point, I guess the previous para then (if/when it relates to the upcoming code block)
 `...will run the mapped component:`